File: WEB-API/WEB-API/Models/Komentar.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace WEB_API.Models
{
    public class Komentar
    {
        public virtual int IdKomentar { get; set; }
        public virtual Blanket Blanket { get; set; }
        public virtual Resenje Resenje { get; set; }
        public virtual DateTime Datum { get; set; }
        public virtual Korisnik Dodao { get; set; }
        public virtual bool PripadaBlanketu { get; set; }
        public virtual string KomentarData { get; set; }

    }
}

File: AB-ArhivaBlanketa/app/src/main/java/com/ab/ssd/ab_arhivablanketa/Klase/Komentar.java
package com.ab.ssd.ab_arhivablanketa.Klase;


import java.io.Serializable;
import java.sql.Timestamp;

public class Komentar implements Serializable {

    public int ID;
    public Blanket blanket ;
    public Resenje resenje;
    public String datum ;
    public Korisnik dodao ;
    public boolean pripadaBlanketu;
    public String komentarData ;

    public Komentar(){}
    public Komentar(int i, Blanket b, Resenje r, String t, Korisnik k, boolean prip, String kom)
    {
        this.ID=i;
        this.blanket=b;
        this.resenje=r;
        this.datum=t;
        this.dodao=k;
        this.pripadaBlanketu=prip;
        this.komentarData=kom;
    }
}
